app/validation/error_messages.py

```diff
@@ -26,4 +26,5 @@
     'DATE_PERIOD_TOO_LARGE': 'Enter a reporting period less than or equal to %(max)s.',
     'SINGLE_DATE_PERIOD_TOO_EARLY': 'Enter a date after %(min)s.',
     'SINGLE_DATE_PERIOD_TOO_LATE': 'Enter a date before %(max)s.',
+    'MUTUALLY_EXCLUSIVE': 'Uncheck %(non_exclusives)s or "%(exclusive)s" to continue',
```

Comment:
In the original trello car, this says `Deselect` ... https://trello.com/c/N74J2IcB/1775-mutually-exclusive-check-boxes

Reply:
Ahh the AC however says Uncheck so ignore this